### DISCUSSION TITLE
How to predict concentrations in ELF（Epithelial Lining Fluid） using MOBI？

### DISCUSSION THREAD

**Discussion by huke58** [Open-Systems-Pharmacology/Forum#1636] [Questions & Problems]
Hello, everyone，
 I want to predict the drug concentration in the ELF(Epithelial Lining Fluid) of lung after an antibiotic is given by injection. 
According to my previous understanding, PK-Sim could not achieve this function. I tried it in MoBi by referring to the published literature (#927 ), but I'm not sure if the structural  and the transporter formula are set up correctly.
I mainly carried out the following steps:
1. ELF subcompartment was created（ **spatial strucues -tree-organism-lung** path）. The ELF related physiological volume and area parameters were input.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/111671487/f59b636c-2970-4968-bd86-974dbfc3d169)

2. Link the blood circulation between ELF subcompartment and cells（ **spatial structrus-diagram** path）.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/111671487/52f89b06-a88a-41f3-b0b9-6e8b234609e7)

3. The permeability parameters of ELF subcompartment were created （**molecules-parameters** path）
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/111671487/71dca519-eb48-4333-b0e8-f30384acacae)

4. Finally, and most importantly, I created a passive transports （ **passive transports**path） to characterize drug transport between cells and ELF.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/111671487/94136e5e-affa-4e77-aa41-b6a792211a6b)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/111671487/6c7fff06-e4b5-43c8-83a6-7db3d58e8ffd)

I don't know if these steps are correct, especially the last formula. I hope you can help me. Thank you in advance

**Comment by WafaaAldhafiri** [comment]
Hello @huke58 , 

Here is this publication that would be able to help you build a pulmonary model and predict ELF Conc. 
https://pubmed.ncbi.nlm.nih.gov/32941997/

Good luck, 
Wafaa
